e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @integrations
+
+describe('Integrations', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testChannelUrl1;
+    let testChannelUrl2;
+    const commandTrigger = 'test-ephemeral';
+    const commandURL = 'http://hidden-peak-21733.herokuapp.com/test_ephemeral';
+    const timestamp = Date.now();
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+                EnableOnlyAdminIntegrations: false,
+            },
+        });
+
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+            testChannelUrl1 = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+
+            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({anotherTeam}) => {
+                team2 = anotherTeam;
+                testChannelUrl2 = `/${team.name}/channels/town-square`;
+                cy.apiAddUserToTeam(team2.id, user1.id);
+                cy.apiAddUserToTeam(team2.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T687 /msg', () => {
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl1);
+
+        // # Post message
+        cy.postMessage(`/msg @${user2.username} Test message`);
+
+        // * The user stays in the same team
+        cy.get(`#${team1.name}TeamButton`).parent().should('have.class', 'active');
+
+        // * The user is in the DM channel with user2
+        cy.get(`#sidebarItem_${Cypress._.sortBy([user1.id, user2.id]).join('__')}`).parent().should('have.class', 'active');
```

Comment:
This line failed on me.
<img width="659" alt="Screen Shot 2020-09-15 at 1 51 27 AM" src="https://user-images.githubusercontent.com/5334504/93120481-fe2cf100-f6f5-11ea-9ecd-b20a2649e14b.png">

Reply:
Some typos with some variables. Not sure why it was working for me before submitting the PR. Will fix.